R/zchunk_LA101.EIA_SEDS.R

```diff
@@ -34,48 +36,68 @@ module_gcam.usa_LA101.EIA_SEDS_DISABLED <- function(command, ...) {
     A_fuel_conv <- get_data(all_data, "gcam-usa/A_fuel_conv")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` and/or 'match' calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Merge-and-Match
+
+    # Define vector to be used for filtering
+    GCAM_year <- 1971:2010
```

Comment:
Is this meant to be the same as `HISTORICAL_YEARS`?  If so, then you should use that constant instead.  If not, then you should explain where this range comes from (i.e., is it determined by the data covered in one or more of the input files or something of that sort?)